silo/forms.py

```diff
@@ -57,6 +60,16 @@ class Meta:
         fields = ['id', 'name', 'description', 'tags', 'shared',
                   'share_with_organization', 'owner', 'workflowlevel1']
 
+    def clean_shared(self):
+        data = self.cleaned_data['shared']
+        ids = [user.pk for user in data]
```

Comment:
Don't loop through it. It returns a QuerySet, so use the `filter()` and `exists()` to check if the owner is in the `data` or not.